alfred-telemetry-solr/README.md

```diff
@@ -0,0 +1,89 @@
+# Solr Prometheus endpoint
+
+This subproject creates the artifacts and the docker images needed for exposing Alfresco-Solr metrics in Prometheus format.
+
+It is implemented for 2 flavors of Solr: solr4 and solr6 (solr libraries changed in some respects).
+
+There are 4 kinds of metrics exposed:
+
+* core stats: information about number of nodes, transactions, acls, change set transactions, errors, states, unindexed nodes
+* FTS metrics: information about full text search
+* tracker metrics: information about last transaction / change set indexed, lag, transactions / change sets remaining
+* jmx metrics: jvm-related metrics (memory, load, GC), Catalina metrics (thread pool, sessions, requests info) for solr4 and solr specific metrics
+
+They can be switch on/off with parameters added to the url:
```

Comment:
That typically not how they configure which metrics to expose. 
@kerkhofsd what do you think about that ?

Reply:
Who is "they"?

Reminder: this implementation does not use micrometer, does not abstract metrics for other endpoints than Prometheus (@thijslemmens). You can see this as a small baby step in a big masterplan.

Reply:
Sure, just wondering that we want to introduce a new paradigma here: now we have a configurable url for scraping

Reply:
Pros: we can get rid of the external prometheus-exporter container and even of the Prometheus agent for solr jvm metrics.

Reply:
Yes, completely agree with the pro's, but I was talking about something else:
Is it a lot more work to use a conf file to switch metrics on/off ?

Reply:
Isn't it more flexible to have a parameter in the url?
Easier to test as well.

Reply:
Con: it complicates configuration of the scraping service?

Reply:
Why does it complicate?
It has custom metrics_path + custom params anyway (wt=prometheus).

Reply:
You mean the metrics are enabled by default, but you can disable some of them with a query-parameter, like `enableFTSMetrics=false` - but using the endpoint `/solr4/alfresco/prometheus?wt=prometheus` works fine ?

That looks cool.